robocop/config.py

```diff
@@ -328,30 +322,11 @@ def load_pyproject_file(self):
         parse_toml_to_config(config, self)
         self.config_from = pyproject_path
 
-    @staticmethod
-    def replace_in_set(container, old_key, new_key):
-        if old_key not in container:
-            return
-        container.remove(old_key)
-        container.add(new_key)
-
     def validate_rule_names(self, rules):
-        deprecated = {
```

Comment:
Instead of removing this dict, maybe we can leave it empty? So that we don't need to come up with the same concept again in the future?

Reply:
Good idea, it may be usefull in the future